pages/fundamentals/introduction.html

```diff
@@ -1,190 +1,185 @@
-﻿<h1>An Introduction of Go</h1>
+﻿<h1>Go 소개</h1>
 
 <p>
-Go is a compiled and static typed programming language born from Google.
-Many of the core Go design and tool development team members have many years
-of experience in the field of programming language research.
+Go는 구글에서 만든 컴파일된 정적 타입 프로그래밍 언어 입니다.
+Go 디자인 및 도구 개발 팀의 많은 핵심 팀원들은 프로그래밍 언어 연구 분야에서 수년간의 경험을 가지고 있습니다.
 </p>
 
 <div>
-Go has many features. Some are unique, some are borrowed from other programming languages:
+Go는 많은 기능이 있습니다. 일부는 고유하게 작성된 것이고 일부는 다른 언어에서 차용하였습니다
 <ul>
 <li>
-	built-in concurrent programming support
+	내장 동시 프로그래밍 지원
 	<ul>
 	<li>
-		goroutines (green threads) and start new goroutines easily.
+		고루틴(녹색 글타래)을 사용하여 새 고루틴을 쉽게 시작할 수 있습니다.
 	</li>
 	<li>
-		channels (based on CSP model) and select mechanisms to do synchronizations between goroutines.
+		채널(CSP 모델 기반) 및 고루틴 간 동기화를 위한 메커니즘 선택

```

Comment:
여기서 `select`는 키워드로 보입니다.

```suggestion
		채널(CSP 모델 기반) 및 고루틴 간 동기화를 위한 <code>select</code> 메커니즘
```

Reply:
수정했습니다